tools/dhtscanner.cpp

```diff
@@ -63,7 +63,7 @@ step(DhtRunner& dht, std::atomic_uint& done, std::shared_ptr<NodeSet> all_nodes,
         done--;
         std::cout << done.load() << " operations left, " << all_nodes->size() << " nodes found." << std::endl;
         cv.notify_one();
-    });
+    }, {}, {});
```

Comment:
@aberaud: can you comment on this? It makes Travis happy, but the [signature](https://github.com/sim590/opendht/blob/queries/include/opendht/dhtrunner.h#L65) it's using should allow empty parameter... I don't really like this as it more looks like a work around the issue.